test/k8s/suite_test.go

```diff
@@ -103,5 +103,6 @@ var _ = ReportAfterSuite("compile report", func(ginkgoReport Report) {
 })
 
 var (
-	_ = Describe("Simple", Simple, Ordered)
+	_ = XDescribe("Simple", Simple, Ordered)
```

Comment:
why are we disabling simple test case?

Reply:
This is a mistake. I forgot why I did this. Probably I was trying to focus on my own case (well, I can actually use `F`)